### PR TITLE
Update docs for alpha 2

### PR DESCRIPTION
**Pull request by neilpa**

### CODE REVIEW

**Comment by neilpa** [comment]
I'm going to YOLO this so I can get the alpha 2 build out

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ improvements from Swift 1.2 to provide a simpler API.
 
 ## Alpha 2
 
+#### Simplified Observer API
```
Comment (NachoSoto) [inline CHANGELOG.md]: I would mention these 2 additions as well: #2439 and #2449

**Comment by NachoSoto** [comment]
> I'm going to YOLO this so I can get the alpha 2 build out

:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (neilpa) [inline CHANGELOG.md]: I'll sneak this in real quick before hitting the release

**Pull request merged**